acqa-repo1-aws-tf12-part1.tf

```diff
@@ -168,9 +168,9 @@ EOF
 # Create lambda function
 resource "aws_lambda_function" "acqa-test-lambda1" {
```

Comment:
**Tenable.cs remediation**
 
**[LOW]: Ensure tracing is enabled for AWS Lambda Functions**
 
**Vulnerability**
Lambda tracing if not enabled can have an impact the integrity and incident response.
 
**Remediation**
In AWS Console -
1. Sign in to AWS Console and go to the Functions page on the Lambda console.
2. Select a function.
3. Select Configuration and then choose Monitoring tools.
4. Select Edit.
5. Under X-Ray, enable Active tracing.
6. Click Save.

In Terraform -
1. In the aws_lambda_function resource, set the tracing_config.mode field to Active.

References:
[https://docs.aws.amazon.com/lambda/latest/dg/services-xray.html](https://docs.aws.amazon.com/lambda/latest/dg/services-xray.html)
[https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_function#tracing_config](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_function#tracing_config)

source: tenablecs-